src/net/yapbam/export/HtmlFormatWriter.java

```diff
@@ -1,21 +1,47 @@
 package net.yapbam.export;
```

Comment:
Maybe we should have css classes attached to the different elements in order to have a complete control of the look & feel in the css (a class for each column, one for column nmae, one for expenses, one for receipts, etc ...).